docs/table-partitioning-bucketing-sort-order-indexes.md

```diff
@@ -125,7 +125,7 @@ Note: If the direction value is `ascending`, the default ordering value is `null
 <TabItem value="java" label="Java">
 
 ```java
-SortOrders.of(FieldReferenceDTO.of("score"), SortDirection.ASCENDING, NullOrdering.NULLS_LAST);
+SortOrders.of(NamedReference.field("score"), SortDirection.ASCENDING, NullOrdering.NULLS_LAST);
```

Comment:
@mchades can you please confirm this?

Reply:
LGTM. It represents a reference to the `score` field.